kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/WatchTest.java

```diff
@@ -276,4 +277,40 @@ private static WatchEvent outdatedEvent() {
         "410: The event in requested index is outdated and cleared (the requested history has been cleared [3/1]) [2]")
       .build()).build();
   }
+
+
+  @Test
+  @DisplayName("TryWithResources, connects and receives event then receives GONE, should receive first event and then close")
```

Comment:
This description seems to be copied from `testTryWithResourcesConnectsThenReceivesEvent` Could you please update this?

https://github.com/fabric8io/kubernetes-client/blob/08261fba6c9306eb063534ec2320e9166f85c71c/kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/WatchTest.java#L65

Reply:
yes, I saw and kept as it is but now modified it as below.

```
  @DisplayName("TryWithResources, connects and receives event then receives GONE, should receive BOOKMARK event and then close it")
```